TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+            cesium: path.resolve(__dirname, cesiumSource)
+        },
+        mainFiles: ['module', 'main', 'Cesium']
+    },
+    module: {
+        rules: [{
+            test: /\.css$/,
+            use: [ 'style-loader', 'css-loader' ]
+        }, {
+            test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
+            use: [ 'url-loader' ]
+        }]
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            template: 'src/index.html'
+        }),
+        // Copy Cesium Assets, Widgets, and Workers to a static directory
+        new CopyWebpackPlugin({ 
+            patterns: [
+                { from: path.join(cesiumSource, cesiumWorkers), to: 'Workers' },
+                { from: path.join(cesiumSource, 'Assets'), to: 'Assets' },
+                { from: path.join(cesiumSource, 'Widgets'), to: 'Widgets' }
+            ]
+        }),
+        new webpack.DefinePlugin({
+            // Define relative base path in cesium for loading assets
+            CESIUM_BASE_URL: JSON.stringify('')
+        })
+    ],
+    mode: 'development',
+};
+```
+
+## Incorporate CesiumJS
+1. Updated `index.js` with CesiumJS starter code:
+```
+import { Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "cesium";
+import "cesium/Widgets/widgets.css";
+import "../src/css/main.css"
+
+// Your access token can be found at: https://cesium.com/ion/tokens.
+// This is the default access token from your ion account
+Ion.defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJlYWE1OWUxNy1mMWZiLTQzYjYtYTQ0OS1kMWFjYmFkNjc5YzciLCJpZCI6NTc3MzMsImlhdCI6MTYyNzg0NTE4Mn0.XcKpgANiY19MC4bdFUXMVEBToBmqS8kuYpUlxJHYZxk';
+
+// Initialize the Cesium Viewer in the HTML element with the `cesiumContainer` ID.
+const viewer = new Viewer('cesiumContainer', {
+  terrainProvider: createWorldTerrain()
+});
+
+// Add Cesium OSM Buildings, a global 3D buildings layer.
+viewer.scene.primitives.add(createOsmBuildings());   
+
+// Fly the camera to San Francisco at the given longitude, latitude, and height.
+viewer.camera.flyTo({
+  destination : Cartesian3.fromDegrees(-122.4175, 37.655, 400),
+  orientation : {
+    heading : Math.toRadians(0.0),
+    pitch : Math.toRadians(-15.0),
+  }
+});
+```
+This code initializes the Cesium Viewer, adds Cesium OSM Buildings to the terrain, and moves the `Camera` to San Francisco. 
+2. Update `index.html` with the CesiumContainer
+```
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+</head>
+<body>
+  <div id="cesiumContainer"></div>
+</body>
+</html>
+```
+3. Run the command `npm run build`. Ensure that the project is built correctly.
+4. Run `npm start` and open http://localhost:8080/ in a web browser to see the CesiumJS viewer.
+
+![South San Francisco](./screenshots/ssf2.png)
+
+Copy and paste your favorite Sandcastle example. For example, [The Particle System Fireworks](https://sandcastle.cesium.com/?src=Particle%20System%20Fireworks.html) demo makes for a fun demo.
```

Comment:
Whats the reasoning behind recommending this as the starting point? I don't think "fun demo" is right.

Reply:
The last version of this tutorial also recommended this demo. I personally like the fireworks demo a lot. It is very simple but also very flashy. Occasionally, I use it to get new community members enthusiastic about CesiumJS. Regardless, I added a commit that tweaks the wording here.

Reply:
Thats completely the reverse way. My point was that you should be suggesting a better fit sandcastle to the reader, not fixing the wording. Please pick a different sandcastle that highlights Cesium's capabilities.